File: java/algorithms/easy/aVeryBigSum.java
/**
 * @author Rithie Natan Carvalhaes Prado
 * @date 2026-06-13
 * Last modified: 2026-06-13
 * @description A Very Big Sum - HackerRank
 * @see {@link https://www.hackerrank.com/challenges/a-very-big-sum/problem}
 */
//---------- Imports ----------
import java.util.List;


//---------- solution ----------
class Result
{
    /**
     * Complete the 'aVeryBigSum' function below.
     * @param {List<Long>} ar
     * @return {long}
     */
    public static long aVeryBigSum(List<Long> ar)
    { return(ar.stream().mapToLong(x -> x).sum()); }
}//end class

File: go/algorithms/easy/aVeryBigSum.go
/*
Author: Rithie Natan Carvalhaes Prado
Date: 2026-06-13
Last modified: 2026-06-13
Description: A Very Big Sum - HackerRank
See: https://www.hackerrank.com/challenges/a-very-big-sum/problem
*/
//--------- package ---------
package easy


//--------- functions ---------
// Complete the aVeryBigSum function below.
func aVeryBigSum(ar []int64) int64 {
	var sum int64 = 0;

	for _, value := range ar {
		sum += value;
	}//end for

    return sum;
}//end aVeryBigSum()